builtin/logical/pki/crl_test.go

```diff
@@ -806,6 +807,7 @@ func TestAutoRebuild(t *testing.T) {
 		LogicalBackends: map[string]logical.Factory{
 			"pki": Factory,
 		},
+		EnableRaw: true,
```

Comment:
Can you share what this is?

Reply:
Ah, it enables the sys/raw endpoint: https://www.vaultproject.io/api/system/raw